File: packages/gunreip/laravel-translation-workbench/resources/views/pages/tw-graph/samples/documentation/idea-to-paper/flow/while/code-examples/while-nested.php
<?php
$groups = loadGroups();
$groupIndex = 0;
while ($groupIndex < count($groups)) {
    $items = $groups[$groupIndex];
    $itemIndex = 0;
    while ($itemIndex < count($items)) {
        processItem($items[$itemIndex]);
        $itemIndex = $itemIndex + 1;
    }
    $groupIndex = $groupIndex + 1;
}
showSummary();

File: packages/gunreip/laravel-translation-workbench/resources/views/pages/tw-graph/samples/documentation/idea-to-paper/flow/while/code-examples/while-nested.java
List<List<Item>> groups = loadGroups();
int groupIndex = 0;
while (groupIndex < groups.size()) {
    List<Item> items = groups.get(groupIndex);
    int itemIndex = 0;
    while (itemIndex < items.size()) {
        processItem(items.get(itemIndex));
        itemIndex = itemIndex + 1;
    }
    groupIndex = groupIndex + 1;
}
showSummary();
